semgrep/semgrep/commands/scan.py

```diff
@@ -138,11 +210,12 @@ def convert(
 @click.option(
     "--severity",
     multiple=True,
-    type=click.Choice(["INFO", "WARNING", "ERROR"]),
+    type=click.Choice([e.value for e in RuleSeverity]),
     help=(
         "Report findings only from rules matching the supplied severity level. By default all applicable rules are run."
-        "Can add multiple times. Each should be one of INFO, WARNING, or ERROR."
+        "Can add multiple times. Each should be one of INFO, WARNING, ERROR, or INVENTORY."
```

Comment:
I think for now we want to keep INVENTORY severity internal instead of something users will ever specify

Reply:
@ajbt200128 mind updating the PR so INVENTORY is not mentioned as an option :D